### DISCUSSION TITLE
How to update id email used with let's encrypt?

### DISCUSSION THREAD

**Discussion by uebmaster** [EasyEngine/easyengine#1575] [Support Requests]
Making a test i've created a wp site using a yopmail email, but i did it just for test, i thinked that deleting the site will revocate the ssl certificate and i will have the oportunity to change the email associated with the let's encrypt certificate, but when i reinstall the site EE just download the certificates and don't let me change the id email.

Searching I found the following command that according to does what I want:

certbot update_account

But when i try to execute certbot in command line i get command not found.

How can i achieve what i want?

Another detail is that i have other sites in the same server, when i reinstall the site mentioned above i can see it, the subdomain redirect me to one of the other sites that i have in the server, but not to the site i just reinstalled, what is happen?

Thanks in advance.

**Comment by kirtangajjar** [comment]
`ee config set le-mail admin@example.com`

**Reply by uebmaster** [reply]
it works (i suppose, i don't know how to check it), but why i can't see the website when i use http:// (not ssl) or www. ?

It's only in that website, cause in the another websites i can see them, althought in the another sites i am using a custom ssl certified.

It can be fixed?
